cost/aws/unused_rds/README.md

```diff
@@ -1,4 +1,4 @@
-# AWS Unused RDS Policy
+# AWS Unused RDS Instances
```

Comment:
👍

Reply:
Done! I only updated the README file without any changelog and version update